doc/man1/openssl-verification-options.pod

```diff
@@ -13,19 +13,22 @@ I<command>
 
 =head1 DESCRIPTION
 
-There are many situations where X.509 certificates are verified
+There are many situations where X.509 public-key certificates are verified
 within the OpenSSL libraries and in various OpenSSL commands.
+For checking certificates, the term I<validation> would acually be more
+adequate, but for historical reasons typically the term I<verification> is used.
 
 Certificate verification is implemented by L<X509_verify_cert(3)>.
-It is a complicated process consisting of a number of steps
+It is a complicated process consisting of many steps
 and depending on numerous options.
 The most important of them are detailed in the following sections.
 
-In a nutshell, a valid chain of certificates needs to be built up and verified
-starting from the I<target certificate> that is to be verified
-and ending in a certificate that due to some policy is trusted.
-Verification is done relative to the given I<purpose>, which is the intended use
-of the target certificate, such as SSL server, or by default for any purpose.
+In a nutshell, a valid chain of certificates needs to be built up and checked
+starting from the I<target certificate> that is to be validated and ending in a
+trust anchor, which usually is a certificate that due to some policy is trusted.
+Certificate validation can be done relative to a given Extended Key Usage (EKU)
+or a given I<purpose>, which is the intended use of the target certificate,
+such as SSL server, or (by default) for any EKU and any purpose.
```

Comment:
The above and other editorial changes, unrelated to support for EKU constraint enforcement belong in a separate PR, please split off.

Reply:
As requested, I removed those (not directly related) changes from this PR
though this will cause merge conflicts when the separate PR(s) will get merged.

Reply:
I found that I already had a PR, namely #18764, where those extra changes to `openssl-verification-options.pod` fit perfectly, so moved them there.

Reply:
> As requested, I removed those (not directly related) changes from this PR though this will cause merge conflicts when the separate PR(s) will get merged.

You can use the current PR as the base revision for the separate PR and indicate that its review should wait for the current one to merged, or just wait to submit it. Whatever works for you.

Reply:
Thank you for these hints.
At that point I had already split the commits to separate PRs,
and as the changes  to `openssl-verification-options.pod` done meanwhile in this PR are confined to a single paragraph, the merge conflicts with #18764 will be bearable.